conversion.go

```diff
@@ -15,6 +15,9 @@ import (
 
 const (
 	maxWords = 256 / bits.UintSize // number of big.Words in 256-bit
+
```

Comment:
Would perhaps be in order to put a comment  here to explain what the code below does